extensions/Khronos/KHR_lights/README.md

```diff
@@ -0,0 +1,144 @@
+
+```javascript
+"scenes" : [
+    {
+        "extensions" : {
+            "KHR_lights" : {
+                "light" : 0
+            }
+        }
+    }            
+]
+```
+
+### Light Types
+
+All light types share the common set of properties listed below.
+
+#### Light Shared Properties
+
+| Property | Description | Required |
+|:-----------------------|:------------------------------------------| :--------------------------|
+| `name` | Name of the light. | No, Default: `""` |
+| `color` | RGB value for light's color in linear space. | No, Default: `[1.0, 1.0, 1.0]` |
+| `intensity` | Brightness of light in. The units that this is defined in depend on the type of light. `point` and `spot` lights use luminous intensity in candela (lm/sr) while `directional` lights use illuminance in lux (lm/m^2) | No, Default: `1.0` |
+| `type` | Declares the type of the light. | :white_check_mark: Yes |
+
+#### Ambient
+
+Ambient lights define constant lighting throughout the scene. They are referenced only by a scene object and only 1 can be referenced per scene.
+
+#### Directional
+
+Directional lights are light sources that act as though they are infinitely far away and emit light in the direction of the +z axis. This light type inherits the orientation of the node that it belongs to. Because it is at an infinite distance, the light is not attenuated. It's intensity is defined in lumens per metre squared, or lux (lm/m^2).
+
+#### Point
+
+Point lights emit light in all directions from a position in space. The brightness of the light attenuates in a physically correct manner as distance increases from the light's position (i.e. brightness goes like the inverse square of the distance). Point light intensity is defined in candela, which is lumens per square radian (lm/sr).
+
+#### Spot
+
+Spot lights emit light in a cone over a distance. The angle and falloff of the cone is defined using two numbers, the `innerConeAngle` and `outerConeAngle`. As with point lights, the brightness also attenuates in a physically correct manner as distance increases from the light's position (i.e. brightness goes like the inverse square of the distance). Spot light intensity refers to the brightness inside the `innerConeAngle` and is defined in candela, which is lumens per square radian (lm/sr). Engines that don't support two angles for spotlights should use `outerConeAngle` as the spotlight angle (leaving `innerConeAngle` to implicitly be `0`).
+
+| Property | Description | Required |
+|:-----------------------|:------------------------------------------| :--------------------------|
+| `innerConeAngle` | Angle, in radians, from centre of spotlight where falloff begins. Must be greater than `0`, less than `outerConeAngle` and less than `PI / 2.0`. | No, Default: `0` |
```

Comment:
Have a copy mistake here — based on default value and paragraph above, there is no requirement that `innerConeAngle` be greater than 0, right?

Reply:
Not sure that I follow. `innerConeAngle` should be greater than 0, shouldn't it?

I do have some redundancy here as I specify that `outerConeAngle` should be greater than 0 and greater than `innerConeAngle` when it would suffice to just mention `innerConeAngle`. Ditto for `innerConeAngle` being less than PI/2.

I'll clean that up.

Reply:
There is an inconsistency here, between `Must be greater than '0'` and `Default: '0'`.

Reply:
Ah, gotcha :)

Reply:
Fixed